duckduckgo/Android
# Update icons and add count to tab icon

Task/Issue URL: https://app.asana.com/0/361428290920652/608322930507369
Tech Design URL: N/A

**Description**:
• Update fire and tab icons to have correct dimensions
• Add count to tab icon

**Steps to test this PR**:
1. Launch the app and launch a few tabs
1. Ensure that the fire and tab icons looks ok and that the correct tab count continues to update during the above
1. Repeat on differnt devices to ensure rendering works across APIs and device densities

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
